Add FAQ (#51)
balena-io-projects/balena-dash: FAQ.md

What hardware do I need?
-------------------------------------------

> We’ve partnered with our friends at PiSupply to make it easy for you to get ahold of all the hardware you need to build this project in a single package. We’ve created two packages, one that includes everything you’ll need, and a second that includes everything apart from the display for if you’ve already got a monitor you want to use.

Reviewer: ```suggestion
We’ve partnered with our friends at PiSupply to make it easy for you to get ahold of all the hardware you need to build this project in a single package. We’ve created two packages, one that includes everything you’ll need, and a second that includes everything apart from the display if you’ve already got a monitor you want to use.
```

- [balenaDash kit](https://uk.pi-supply.com/products/balenadash-kit)
- [balenaDash kit - barebones](https://uk.pi-supply.com/products/balenadash-kit-barebones)

The list of items you’ll need is also included below:

> - Raspberry Pi 3B/3B+ (Note: this project will not work with the Pi Zero or older devices with < 1GB RAM)

Reviewer: ```suggestion
- Raspberry Pi 3B/3B+ (Note: this project will not work with Pi Zero or older devices with < 1GB RAM)
```

- 16GB Micro-SD Card (we recommend Sandisk Extreme Pro SD cards)
- Display (any Raspberry Pi display will work for this project)
- Micro-USB cable
- Power supply
- Case (optional)

> Can you use this balena-dash without the Raspberry Pi display?

Reviewer: ```suggestion
Can you use this balenaDash without the Raspberry Pi display?
```
